dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/view/Page.java

```diff
@@ -139,5 +152,7 @@ public static class Pager {
     @JsonProperty private Integer pageSize;
     @JsonProperty private Long total;
     @JsonProperty private Integer pageCount;
+    @JsonProperty private String prev;
```

Comment:
The names have to match the https://github.com/dhis2/dhis2-core/blob/40e950822f2c1346518958022fc22fe61e07f959/dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/Pager.java#L47-L49

`https://play.im.dhis2.org/dev/api/organisationUnits?page=2&pageSize=2&fields=displayName`

```json
{
  "pager": {
    "page": 2,
    "total": 1332,
    "pageSize": 2,
    "nextPage": "https://play.im.dhis2.org/dev/api/organisationUnits?page=3&pageSize=2&fields=displayName",
    "prevPage": "https://play.im.dhis2.org/dev/api/organisationUnits?pageSize=2&fields=displayName",
    "pageCount": 666
  },
  "organisationUnits": [
    {
      "displayName": "Agape CHP"
    },
    {
      "displayName": "Ahamadyya Mission Cl"
    }
  ]
}
```

We could add a unit test to compare JSON produced by common.Pager and our view.Page as a safety-net until we can reuse the common.Pager.